powsybl-parent/pom.xml

```diff
@@ -352,5 +355,73 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>lombok-config</id>
+            <!-- Add at the start of lombok.config this line: import target/configs/powsybl-build-tools.jar!powsybl-build-tools/lombok.config -->
+            <activation>
+                <property>
+                    <name>powsybl.lombok.config</name>
+                    <value>true</value>
+                </property>
+                <!--<file>
+                    <exists>lombok.config</exists>
+                </file>-->
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>lombok-copy-deps</id>
+                                <phase>initialize</phase>
+                                <goals>
+                                    <goal>copy</goal>
+                                </goals>
+                                <configuration>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>com.powsybl</groupId>
+                                            <artifactId>powsybl-build-tools</artifactId>
+                                            <version>${powsybl-build-tools.version}</version>
+                                            <type>jar</type>
+                                            <overWrite>false</overWrite>
+                                            <outputDirectory>${project.build.directory}/configs</outputDirectory>
+                                            <destFileName>powsybl-build-tools.jar</destFileName>
+                                        </artifactItem>
+                                    </artifactItems>
+                                    <outputDirectory>${project.build.directory}/deps</outputDirectory>
+                                    <overWriteReleases>false</overWriteReleases>
+                                    <overWriteSnapshots>true</overWriteSnapshots>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
```

Comment:
can we have this in the "checks" profile/property so that it may be turned off like other checks?

Reply:
done in commit with marker files